src/strategies/lodestar-vesting/index.ts

```diff
@@ -0,0 +1,49 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+import { getAddress } from '@ethersproject/address';
+
+export const author = '0xAppo';
+export const version = '0.1.0';
+
+const abi = [
+  'function balanceOf(address account) external view returns (uint256)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+if (
+      options.beneficiaryAddresses.length >= 25 ||
+      options.contractAddresses.length >= 25
+    ) {
+      console.error('ERROR: Too many stake pool addresses provided.');
+      return;
```

Comment:
Should throw an error like here:
https://github.com/snapshot-labs/snapshot-strategies/blob/f4c2302101269fa27e3c71fe9bc7b3f204ee24f2/src/strategies/math/options.ts#L70